### PR TITLE
Development

### PR DESCRIPTION
**Pull request by IBTechRaj**

### CODE REVIEW

File: re-former/config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+
+	resources :users
```
Comment (azdrenymeri) [inline re-former/config/routes.rb]: you should cover only `:new` ,`:create`,`:edit`,`:update`

File: re-former/app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
+class UsersController < ApplicationController
+	def new
+	@user = User.new
+	end
+
+	def create
+#@user = User.new(username: params[:username], email: params[:email], password: params[:password])
```
Comment (azdrenymeri) [inline re-former/app/controllers/users_controller.rb]: Remove this comment

File: re-former/app/views/users/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+<%#= text_field_tag( :email) %>
+<%#= label_tag('User name : ') %>
+<%#= text_field_tag( :username) %>
+<%#= label_tag('password  : ') %>
+<%#= password_field_tag( :password) %>
+<%#= submit_tag('submit ') %>
+<%#= end %>
+
+<%= form_for @user, url: {action: "create"}  do |f| %>
+  Email  : <%= f.email_field :email , placeholder: 'example@example.com'%><br />
```
Comment (azdrenymeri) [inline re-former/app/views/users/new.html.erb]: you don't need to explicitly type Email: you can use `f.label :email`,`f.label :username`,`f.label :password`

File: re-former/config/routes.rb (hunk 1 above)
Comment (IBTechRaj) [reply re-former/config/routes.rb]: Modified as advised

File: re-former/app/controllers/users_controller.rb (hunk 2 above)
Comment (IBTechRaj) [reply re-former/app/controllers/users_controller.rb]: That comment is removed.

File: re-former/app/views/users/new.html.erb (hunk 3 above)
Comment (IBTechRaj) [reply re-former/app/views/users/new.html.erb]: Changed the code accordingly.

**Comment by IBTechRaj** [comment]
We have made all the corrections suggested.
Also, ran the command 'rake db:migrate:status' to check for pending migrations and got the status 'up'.  Please let us know about that 'pending migration'

**Comment by azdrenymeri** [comment]
## Hello @IBTechRaj and @DanielMitiku 
#### Good job on implementing the requested changes
#### There are no further requests from me :+1:

**Pull request merged**